server/src/infra/migrations/1700747853368-AddAssetFaceIndicies.ts

```diff
@@ -0,0 +1,17 @@
+import { MigrationInterface, QueryRunner } from "typeorm"
+
+export class AddAssetFaceIndicies1700747853368 implements MigrationInterface {
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(`CREATE INDEX IF NOT EXISTS IDX_asset_faces_on_assetId ON "asset_faces"("assetId");`);
+    await queryRunner.query(`CREATE INDEX IF NOT EXISTS IDX_asset_faces_on_personId_assetId ON "asset_faces" ("personId", "assetId");`);
```

Comment:
The above two indexes could be combined if the query WHERE clauses are ordered in the correct way. If we always query with personId after assetId then you could just have a single index. PostgreSQL will use the assetId,personId index for any query just involving assetId.